src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -372,7 +376,8 @@ const ActionsPane = (props: {contentHeight: string}) => {
                     onClick={applySelection}
                     disabled={
                       (!selectedResourceId && !selectedPath) ||
-                      (selectedResource && isKustomizationPatch(selectedResource))
+                      (selectedResource &&
+                        (isKustomizationPatch(selectedResource) || !knownResourceKinds.includes(selectedResource.kind)))
```

Comment:
what about the Diff button!?

Reply:
that's a bit above but it's hidden by default in the changes shown by github
![image](https://user-images.githubusercontent.com/20538711/151168384-b755c81a-8498-4124-b076-161f04fbd24c.png)